src/commonMain/kotlin/exchange.dydx.abacus/state/internalstate/InternalState.kt

```diff
@@ -57,6 +58,9 @@ internal data class InternalMarketState(
 
     // grouped orderbook for app display
     var groupedOrderbook: MarketOrderbook? = null,
+
+    // candles: resolution -> candles
+    var candles: MutableMap<String, List<MarketCandle>>? = null
```

Comment:
Should this be `MutableMap<String, MutableMap<String, List<MarketCandle>>>`? Market -> Resolution -> Candles

Reply:
The parent object is keyed by "MarketId".